docs/reference/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -133,25 +155,30 @@ index exceeds 25GB.
 [[apply-policy-manually]]
 === Apply lifecycle policy manually
 
-When you create an index, you can apply a lifecycle policy 
-by specifying the `index.lifecycle.name` setting.
-This causes {ilm-init} to immediately start managing the index.
+When you apply a policy to an existing index through {kib} Management,
+or specify a policy when you create an index, 
+{ilm-init} immediately starts managing the index.
+
+IMPORTANT: Do not manually apply a policy that uses the rollover action.
+Policies that use rollover must be applied by the <<apply-policy-template, index template>>. 
+Otherwise, the policy is not carried forward when the rollover action creates a new index.
 
+[%collapsible]
+.API example
+====
```

Comment:
It's not immediately clear what this example is meant to demonstrate.
It made sense with the previous text but not with the current text.